webapp/components/wpt-results.html

```diff
@@ -63,6 +64,26 @@
       .info small {
         float: right;
       }
+      .top {
+        background-color: #eee;
+      }
+      .passing-0 { background-color: hsl(0, 85%, 65%); }
+      .passing-20 { background-color: hsl(0, 85%, 70%); }
+      .passing-40 { background-color: hsl(0, 85%, 75%); }
+      .passing-60 { background-color: hsl(0, 85%, 80%); }
+      .passing-80 { background-color: hsl(0, 85%, 85%); }
+      .passing-100 { background-color: hsl(129, 85%, 65%); }
+
+      span.delta.negative {
+        color: var(--paper-red-700);
+      }
+      span.delta.positive {
+        color: var(--paper-green-700);
+        font-weight: bold;
```

Comment:
I found it a bit odd that only the green bits are bold. Is this intentional?

Also, could some kind of background be given to the non-empty cells so that they stand out a bit more when scrolling?

Finally, when there aren't any differences at all, should that be highlighted in some way? Otherwise I'd just think the view is broken :)

Reply:
De-boldened, grey'd the background, but didn't do anything for no differences - not likely, and confident the user will notice that all the numbers are the same.